### PR TITLE
Add padding support to LazyBlock

### PR DESCRIPTION
**Pull request by abarth**
Also fill in some related dartdocs.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/lib/src/material/tooltip.dart [hunk 1]
```diff
@@ -63,6 +63,9 @@ class Tooltip extends StatefulWidget {
 
   final double height;
 
+  /// The amount of space to inset the child.
```
Comment (Hixie) [inline packages/flutter/lib/src/material/tooltip.dart]: `by which to inset` ?

File: packages/flutter/lib/src/widgets/lazy_block.dart [hunk 2]
```diff
@@ -77,6 +80,34 @@ class LazyBlockBuilder extends LazyBlockDelegate {
   bool shouldRebuild(LazyBlockDelegate oldDelegate) => true;
 }
 
+/// Uses an [List<Widget>] to provide children for [LazyBlock].
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/lazy_block.dart]: a, not an

File: packages/flutter/lib/src/widgets/lazy_block.dart [hunk 3]
```diff
@@ -77,6 +80,34 @@ class LazyBlockBuilder extends LazyBlockDelegate {
...
+  }
+
+  /// The widgets to display.
+  ///
+  /// This list must not be modified after being stored in this field.
+  final List<Widget> children;
+
+  @override
+  Widget buildItem(BuildContext context, int index) {
+    return index < children.length ? children[index] : null;
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/lazy_block.dart]: `assert(index >= 0)`

**Comment by Hixie** [comment]
LGTM

**Pull request merged**